src/background/stores/dev-tools-store.ts

```diff
@@ -19,6 +19,7 @@ export class DevToolStore extends BaseStoreImpl<DevToolState> {
     public getDefaultState(): DevToolState {
         const defaultValues: DevToolState = {
             isOpen: false,
+            count: 0,
```

Comment:
does it make sense to call this `eTag` and add a comment to clarify why do we need it?

Reply:
I think [eTag](https://en.wikipedia.org/wiki/HTTP_ETag) is not quite the right concept (it's about not doing work when the same page is requested), but agree that it would be good to have something more explicit than `count`. Maybe something like `inspectRequestId` or `distinctInspectId` or something like that?

Reply:
Updated